src/cmake/ov_parallel.cmake

```diff
@@ -331,6 +331,9 @@ function(ov_set_threading_interface_for TARGET_NAME)
     elseif (THREADING STREQUAL "OMP")
         if (WIN32)
             set(omp_lib_name libiomp5md)
+        elseif (RISCV64)
+            message(WARNING "GCC OpenMP will be used on the current platform")
+            set(omp_lib_name gomp)
```

Comment:
we likely need to adjust the logic below, because it still tries to find some libs and define OMP_LIBRARIES_RELEASE variables

Reply:
BTW, have you tried to build TBB using RISC-V toolchain and pass TBB_DIR to OpenVINO cmake? 
maybe it's time to request RISC-V support in TBB

Reply:
Discussed offline: Ilya provided PR with generic OpenMP support on different platforms 
https://github.com/openvinotoolkit/openvino/pull/26175

Thanks a lot! ❤️